pulp_smash/utils.py

```diff
@@ -86,6 +86,36 @@ def os_is_f27(cfg, pulp_host=None):
     return response.returncode == 0
 
 
+def fips_is_supported(cfg, pulp_host=None):
+    """Return ``True`` if the server has crypto fips option available or ``False`` otherwise
+    :param cfg: Information about the system being targeted
+    :param pulp_host: A :class : `pulp_smash.config.PulpHost` to target, instead of the default chosen by :class: pulp_smash.cli.Client`.
+    :return: True of False
+    """
+    try:
+       cli.Client(cfg, cli.echo_handler, pulp_host).run((
+            'sysctl',
+            'crypto.fips_enabled'
+        ))
+    except Exception:
+        return False
+    return True
+
+
+def fips_is_enabled(cfg, pulp_host=None):
+    """Return ``True`` if the server has crypto fips enabled or ``False`` otherwise
+    :param cfg: Information about the system being targeted
+    :param pulp_host: A :class : `pulp_smash.config.PulpHost` to target, instead of the default chosen by :class: pulp_smash.cli.Client`.
+    :return: True of False
+    """
+    response = cli.Client(cfg, cli.echo_handler, pulp_host).run((
+        'sysctl',
+        '--values',
+        'crypto.fips_enabled'
+    ))
+    return response.returncode == 0 and response.stdout.strip() == '1'
```

Comment:
Don't bother checking `response.returncode`. Let the callback response handler worry about checking the response return code. As an aside, this makes it possible to drop the `response` variable and implement this `fips_is_enabled` as a single expression.